### PR TITLE
Translations from Crowdin - Initial import

### PR DESCRIPTION
**Pull request by teolemon**
Enjoy!

### CODE REVIEW

File: po/common/de.po [hunk 1]
```diff
@@ -1,1295 +1,1219 @@
...
+"Language: de_DE\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Crowdin-Project: openfoodfacts\n"
+"X-Crowdin-Language: de\n"
+"X-Crowdin-File: /master/po/common/en.po\n"
 
 #. put here the language tag ("fr" for French, "ja" for Japanese, etc)
 msgctxt ":langtag"
 msgid ":langtag"
-msgstr "de"
```
Comment (hangy) [inline po/common/de.po]: Existing translations removed, same for the rest of the file.

File: po/common/de.po [hunk 2]
```diff
@@ -1,1295 +1,1219 @@
...
-"Language-Team: \n"
+"PO-Revision-Date: 2017-04-08 13:41-0400\n"
+"Last-Translator: pierreslamich <pierre.slamich@gmail.com>\n"
+"Language-Team: German\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "X-Launchpad-Export-Date: 2017-01-18 10:56+0000\n"
-"X-Generator: Poedit 1.8.12\n"
-"Language: de\n"
```
Comment (hangy) [inline po/common/de.po]: Language doesn't match filename any more. Is this important?

Comment (teolemon) [inline po/common/de.po]: yes, it's doing shit. The Merge Request was auto created. Please don't merge until I fix it.

File: po/common/de.po (hunk 1 above)
Comment (teolemon) [inline po/common/de.po]: yes, it's doing shit. The Merge Request was auto created. Please don't merge until I fix it.

**Pull request closed without merging**